src/machinetalk/support/unionread.c

```diff
@@ -47,6 +46,7 @@ bool print_container(pb_istream_t *stream)
     tag = taghdr >> 3;
     wiretype = taghdr & 0x07;
 #else
+    bool eof;
```

Comment:
Presumably this is to get rid of an unused variable warning?

To my mind the whole \#if \#else construct is meaningless  as soon as `#if 1` is written.
It should be replaced with the code under \#else being in a commented section for reference if of use.